### DISCUSSION TITLE
Search Field

### DISCUSSION THREAD

**Discussion by abuzer** [rnmapbox/maps#3648] [General]
Hi, Is there any search field component available to search for the location? Similar to Google Address auto complete?

**Comment by chriscoderdr** [comment]
I don't think there one in this react native project but they do have the apis for this that are implemented in the reactjs part:

https://docs.mapbox.com/mapbox-search-js/guides/search/react/

in theory you could implement your own using:
https://docs.mapbox.com/mapbox-search-js/guides/#search-js-core
